.browserslistrc

```diff
@@ -9,60 +9,98 @@ node 14
 # `npx browserslist --mobile-to-desktop "> 0.5%, last 2 versions, Firefox ESR, not dead, not IE 11"` when the last major is released.
 # On update check all #stable-snapshot markers
 [stable]
-and_chr 85
-and_ff 80
+and_chr 92
+and_chr 91
+and_ff 90
+and_ff 89
 and_qq 10.4
 and_uc 12.12
-android 85
+android 92
+android 91
 baidu 7.12
-chrome 84
-edge 85
+chrome 92
+chrome 91
+chrome 90
+edge 92
+edge 91
+firefox 90
+firefox 89
 firefox 78
-ios_saf 12.2
+ios_saf 14.5-14.7
+ios_saf 14.0-14.4
+ios_saf 13.4-13.7
 kaios 2.5
 op_mini all
-op_mob 70
-opera 70
-safari 13.1
-samsung 11.1-11.2
+op_mob 77
+op_mob 76
+opera 77
+opera 76
+safari 14.1
+safari 14
+samsung 14.0
+samsung 13.0
 
 # Same as `stable` but with IE 11
 [legacy]
-IE 11
-and_chr 85
-and_ff 80
+ie 11
+and_chr 92
+and_chr 91
+and_ff 90
+and_ff 89
 and_qq 10.4
 and_uc 12.12
-android 85
+android 92
+android 91
 baidu 7.12
-chrome 84
-edge 85
+chrome 92
+chrome 91
+chrome 90
+edge 92
+edge 91
+firefox 90
+firefox 89
 firefox 78
-ios_saf 12.2
+ios_saf 14.5-14.7
+ios_saf 14.0-14.4
+ios_saf 13.4-13.7
 kaios 2.5
 op_mini all
-op_mob 70
-opera 70
-safari 13.1
-samsung 11.1-11.2
+op_mob 77
+op_mob 76
+opera 77
+opera 76
+safari 14.1
+safari 14
+samsung 14.0
+samsung 13.0
 
 # snapshot of `npx browserslist "maintained node versions"`
 # On update check all #stable-snapshot markers
 [node]
-node 12.0
```

Comment:
This can stay.

Reply:
So just keep it as `node 12.0` since that is the lowest version?

Reply:
Yep